app/src/main/java/com/keystone/cold/ui/fragment/main/EthBroadcastTxFragment.java

```diff
@@ -75,10 +75,11 @@ protected void initData(Bundle savedInstanceState) {
     @Override
     public String getSignedTxData() {
         try {
-            JSONObject signed = new JSONObject(txEntity.getSignedHex());
-            signed.remove("abi");
-            signed.remove("chainId");
-            return Hex.toHexString(signed.toString().getBytes(StandardCharsets.UTF_8));
+            JSONObject messageSignatureJson = new JSONObject(messageSignature);
+            byte[] signature = Hex.decode(messageSignatureJson.getString("signature"));
+            byte[] requestId = Hex.decode(messageSignatureJson.getString("signId"));
```

Comment:
rename signId to requestId, And signId is string format of UUID, use UUID to parse it to bytes